dockerizing-notes.md

```diff
@@ -0,0 +1,15 @@
+# Dockerizing Notes
+
+## Running
+
+1. `docker-compose build`
+1. `docker-compose run app npm install --no-shrinkwrap`
+  because the shrinkwrap has darwin-specific packages listed
```

Comment:
Hmm, this is interesting, I thought we had removed those. I'm able to `npm install` on ubuntu. CircleCI also doesn't have any trouble with it. Wonder if npm3 takes care of it?